fusionfusion/annotationFunction.py

```diff
@@ -271,4 +271,28 @@ def merge_fusion_result(input_dir, output_file_path):
     hOUT.close()
 
     subprocess.check_call(["rm", "-rf", output_file_path + ".unsorted.tmp"])
+
+    trace_files = list(filter(os.path.exists, [
+        input_dir + "/star.chimeric.trace.txt",
+        # input_dir + "/ms2.chimeric.trace.txt",
+        # input_dir + "/th2.chimeric.trace.txt"
+    ]))
+    if trace_files:
+        traced_file_path = re.sub('(?:\.txt)?$', '.traced.txt', output_file_path, count=1)
+        emit_traced(output_file_path, trace_files, traced_file_path)
  
+def emit_traced(input_file, trace_files, output_file):
```

Comment:
2 blank lines between top-level function definition
https://www.python.org/dev/peps/pep-0008/#blank-lines

```suggestion
 
 
def emit_traced(input_file, trace_files, output_file):
```